### PR TITLE
Update theme.md - creation of wwwroot folder was missing

### PR DESCRIPTION
**Pull request by MarGraz**
To add a thumbnail for the new template, the user must create a wwwroot folder in the template library project. This step was missing.

### CODE REVIEW

**Comment by MarGraz** [comment]
@dotnet-policy-service agree company="CodingOps"

File: src/docs/getting-started/theme.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ In this article, we are going to create an Orchard Core Theme by adding it to an
 - In Visual Studio Code or CLI, execute the command `dotnet sln add MyTheme.OrchardCore.csproj` to add project to solution. Then, go to `OrchardCore.Cms.Web` folder and execute the command `dotnet add OrchardCore.Cms.Web.csproj reference ../OrchardCore.Themes/MyTheme.OrchardCore.csproj`
 - Set the main Orchard Core CMS Web application as the startup project.
 
-A thumbnail can also be created by adding a `Theme.png` in the `wwwroot` folder.
+Since `MyTheme.OrchardCore` is a library project and doesn't include a `wwwroot` folder by default, you'll need to create one. To add a thumbnail to `MyTheme.OrchardCore` template, you can place a `Theme.png` file inside the `wwwroot` folder.
```
Comment (Skrypt) [inline src/docs/getting-started/theme.md:14]:
```suggestion
A thumbnail can also be used by the Admin UI by adding a `wwwroot\Theme.png` folder/file in your theme project root folder.
```

Comment (MarGraz) [reply src/docs/getting-started/theme.md:14]: @Skrypt thank you

Comment (Skrypt) [reply src/docs/getting-started/theme.md:14]: Ok but you decided to not commit my suggestion?

Comment (MarGraz) [reply src/docs/getting-started/theme.md:14]: @Skrypt I think I clicked the wrong button 😅

**Review by Skrypt — APPROVED** [review]

File: src/docs/getting-started/theme.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ In this article, we are going to create an Orchard Core Theme by adding it to an
 - In Visual Studio Code or CLI, execute the command `dotnet sln add MyTheme.OrchardCore.csproj` to add project to solution. Then, go to `OrchardCore.Cms.Web` folder and execute the command `dotnet add OrchardCore.Cms.Web.csproj reference ../OrchardCore.Themes/MyTheme.OrchardCore.csproj`
 - Set the main Orchard Core CMS Web application as the startup project.
 
-A thumbnail can also be created by adding a `Theme.png` in the `wwwroot` folder.
+A thumbnail can also be used by the Admin UI by adding a `wwwroot\Theme.png` folder/file in your theme project root folder.
```
Comment (MikeAlhayek) [inline src/docs/getting-started/theme.md:14]:
```suggestion
 - The Admin UI can utilize a thumbnail by incorporating a `wwwroot\Theme.png` file within the root folder of the theme project.
```

**Review by MikeAlhayek — CHANGES_REQUESTED** [review]

**Pull request merged**